src/client.cpp

```diff
@@ -340,32 +339,12 @@ PVR_ERROR GetChannels(ADDON_HANDLE handle, bool bRadio)
   return PVR_ERROR_SERVER_ERROR;
 }
 
-bool OpenLiveStream(const PVR_CHANNEL &channel)
```

Comment:
@ksooo to clarify, there are currently three ways for PVR addons to provide video to Kodi:

1. By returning a URL in `GetLiveStreamUrl()`. This means Kodi will just open that URL and handle everything playback related itself. Simple addons such as this one use this mode.

2. `OpenLiveStream`/`CloseLiveStream`/`ReadLiveStream` etc. along with `pCapabilities->bHandlesInputStream = true;`. In this mode Kodi kind of treats the PVR addon as a file handle and the addon itself is responsible for reading the video data. Addons that do client-side timeshifting use this mode.

3. `DemuxRead`, `DemuxClose` etc. methods along with `pCapabilities->bHandlesDemuxing = true;`. Only pvr.hts and pvr.vdr.vnsi use this mode AFAIK (perhaps pvr.mythtv too these days). You probably know tihs already but in this mode Kodi receives demuxed streams directly from the addon and just feeds them to their respective codecs.